src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2127,5 +2127,94 @@
   "com.minecolonies.building.largequarry": "Large Quarry",
   "com.minecolonies.coremod.gui.quarry.desc": "To start the Quarry construct a Mine, hire a Quarrier and assign the Quarrier to the Quarry",
   "com.minecolonies.coremod.gui.workerhuts.druid": "Druid",
-  "item.minecolonies.mistletoe": "Mistletoe"
+  "item.minecolonies.mistletoe": "Mistletoe",
+
+  "minecolonies:builder.job.desc": "Constructs buildings and decorations.",
+  "minecolonies:deliveryman.job.desc": "Delivers requested items and blocks to players and colonists.",
+  "minecolonies:miner.job.desc": "Mines out a shaft system under the colony, gathering blocks and ores. Can be a dangerous Job",
```

Comment:
"Job" in lowercase

Reply:
#inconsistent-ray put a period at the end of it, too!